### PR TITLE
chore: remove todo's from code

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
This removes all todo's from all rs files we have. 
Outstanding valid todo's have tracking issues either in the Tari repo or the dan repo. 

Motivation and Context
---
We should have to relay on obscure hidden todo's in code for future work or incomplete code.

### CODE REVIEW

File: base_layer/core/src/transactions/transaction_components/transaction_output.rs [hunk 1]
```diff
@@ -323,9 +323,6 @@ impl TransactionOutput {
             .as_ref()
             .and_then(|f| f.validator_node_registration())
         {
-            // TODO(SECURITY): Signing this with a blank msg allows the signature to be replayed. Using the commitment
-            //                 is ideal as uniqueness is enforced. However, because the VN and wallet have different
-            //                 keys this becomes difficult. Fix this once we have decided on a solution.
```
Comment (brianp) [inline base_layer/core/src/transactions/transaction_components/transaction_output.rs:328]: 😬

File: base_layer/core/src/validation/aggregate_body/aggregate_body_chain_validator.rs [hunk 2]
```diff
@@ -191,9 +191,6 @@ fn check_validator_node_registration_utxo(
             });
         }
 
-        // TODO(SECURITY): Signing this with a blank msg allows the signature to be replayed. Using the commitment
-        //                 is ideal as uniqueness is enforced. However, because the VN and wallet have different
-        //                 keys this becomes difficult. Fix this once we have decided on a solution.
```
Comment (brianp) [inline base_layer/core/src/validation/aggregate_body/aggregate_body_chain_validator.rs:196]: 😬

File: base_layer/core/src/validation/aggregate_body/aggregate_body_internal_validator.rs [hunk 3]
```diff
@@ -393,9 +393,6 @@ fn check_validator_node_registration_utxo(
             });
         }
 
-        // TODO(SECURITY): Signing this with a blank msg allows the signature to be replayed. Using the commitment
-        //                 is ideal as uniqueness is enforced. However, because the VN and wallet have different
-        //                 keys this becomes difficult. Fix this once we have decided on a solution.
```
Comment (brianp) [inline base_layer/core/src/validation/aggregate_body/aggregate_body_internal_validator.rs:398]: 😬

**Review by brianp — APPROVED** [review]
Ack

Those empty sig todos though...

File: base_layer/core/src/validation/aggregate_body/aggregate_body_internal_validator.rs (hunk 3 above)
Comment (SWvheerden) [reply base_layer/core/src/validation/aggregate_body/aggregate_body_internal_validator.rs:398]: haha yeah, they are now marked as an issue on the dan project to figure out what exactly they want to add there

File: base_layer/core/src/validation/aggregate_body/aggregate_body_chain_validator.rs (hunk 2 above)
Comment (SWvheerden) [reply base_layer/core/src/validation/aggregate_body/aggregate_body_chain_validator.rs:196]: haha yeah, they are now marked as an issue on the dan project to figure out what exactly they want to add there

File: base_layer/core/src/transactions/transaction_components/transaction_output.rs (hunk 1 above)
Comment (SWvheerden) [reply base_layer/core/src/transactions/transaction_components/transaction_output.rs:328]: haha yeah, they are now marked as an issue on the dan project to figure out what exactly they want to add there

**Pull request merged**